### PR TITLE
Car wash

### PR DESCRIPTION
**Pull request by melindamariapop**

### CODE REVIEW

File: app/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -64,6 +74,18 @@ services_section:
     text2: 'Your car is safe with us. <span><b>Parking<span class="main-green">OK</span></b></span> has surveillance cameras and security 24 hours/day, 7 days/week. Our fenced and well lighted parking lot is located next to the airport’s runway which raises the security level by having only one access point, through our check point.'
     text3: "We offer free around-the-clock shuttle service. The shuttle to/from the airport takes about one minute. Upon your return just give us a call from the terminal and the shuttle will be on the way to pick you up."
 
+car_wash_section:
+    intro: "Spalatorie auto chiar langa parcare!"
```
Comment (deepmikoto) [inline app/Resources/translations/messages.en.yml]: Please change to EN

File: app/Resources/views/default/index.html.twig [hunk 2]
```diff
@@ -87,7 +88,7 @@
         <div class="text-center">
             <div class="col-sm-8 col-sm-offset-2">
                 <h2 class="title-one">{{ 'about_as'|trans }}</h2>
-                <p>
+                <p style="font-size: 16px">
```
Comment (deepmikoto) [inline app/Resources/views/default/index.html.twig]: ei ei :smile:

File: src/AdminBundle/Form/ReservationEditType.php [hunk 3]
```diff
@@ -87,6 +88,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ])
                 ]
             ])
+            ->add('carWash', ChoiceType::class, [
+                'placeholder' => 'car_wash_label',
+                'choices' => [
+                    'AUTO_FULL' => 'car_wash_full_auto',
```
Comment (deepmikoto) [inline src/AdminBundle/Form/ReservationEditType.php]: You don't have to change it now, but in the future you can add this array ro a static function somewhere, you have the same one, copy paste in 5 places

File: app/Resources/translations/messages.en.yml (hunk 1 above)
Comment (melindamariapop) [reply app/Resources/translations/messages.en.yml]: nu il folosesc, trebuia sa sterg cheia asta

File: src/AdminBundle/Form/ReservationEditType.php (hunk 3 above)
Comment (melindamariapop) [reply src/AdminBundle/Form/ReservationEditType.php]: I know :)

**Pull request merged**